test/integration/fetchcontent_compatibility.rb

```diff
@@ -0,0 +1,44 @@
+require_relative './lib'
+
+# Tests FetchContent overriding with CPM
+
+class SourceCache < IntegrationTest
```

Comment:
Paste remnant. Class should be called `FetchContentCompatibility` or something like that